shiny/ui/_sidebar.py

```diff
@@ -79,14 +83,27 @@ class SidebarOpen:
     )
 
     @staticmethod
-    def _values_str() -> str:
-        return f"""'{"', '".join(SidebarOpen._VALUES)}'"""
+    def _values_str(values: tuple[str, ...] | None = None) -> str:
+        quoted = [f'"{x}"' for x in (values or SidebarOpen._VALUES)]
+        ret = ""
+        for i, value in enumerate(quoted):
+            if i > 0 and len(quoted) > 2:
+                ret += ","
+            if i == len(quoted) - 1:
+                ret += " or"
+            if i > 0:
+                ret += " "
+            ret += value
+        return ret
```

Comment:
This is a fair amount of complexity to add for formatting of a pretty specific error.

Given the timing of this (the week of release) and that the PR covers a handful of different PRs (making it difficult to review and come back to), I'd prefer to leave this as a follow up item.

Also, somewhat relatedly, there are other diffs to add line breaks to error messages. I guess I care less about those, but anything we can do to keep the diff minimal would be great.